utils/sccli.py

```diff
@@ -55,6 +58,48 @@ def system(cmd):
     returncode = ret.returncode
     return out, err, returncode
 
+def get_all_interface_ip():
+    interface_list = ''
+    for interface in netifaces.interfaces():
+        addr = netifaces.ifaddresses(interface).get(netifaces.AF_INET)
+        if addr:
+            interface_list += "%s," % addr[0].get('addr')
+    return interface_list
+
+def set_proxy():
+    proxy = os.getenv('PROXY', '')
+    if proxy:
+        interface_ip_list = get_all_interface_ip()
+        predefine_no_proxy_list = ".xip.io,172.30.0.0/16,172.17.0.0/16,%s" % socket.gethostname()
```

Comment:
Should that not use the SUB_DOMAIN variable in some form? Hard coding _xip.io_ looks wrong here at the first glance. Am I missing something?